src/scrapers/mizrahi.ts

```diff
@@ -124,14 +129,22 @@ async function extractPendingTransactions(page: Page): Promise<Transaction[]> {
   });
 }
 
+async function postLogin(page: Page) {
+  await Promise.race([
+    waitUntilElementFound(page, afterLoginSelector),
+    waitUntilElementFound(page, invalidPasswordSelector),
+  ]);
+}
+
 class MizrahiScraper extends BaseScraperWithBrowser {
   getLoginOptions(credentials: ScraperCredentials) {
     return {
-      loginUrl: `${LOGIN_URL}`,
+      loginUrl: LOGIN_URL,
       fields: createLoginFields(credentials),
-      submitButtonSelector: '#ctl00_PlaceHolderLogin_ctl00_Enter',
-      postAction: async () => waitForNavigation(this.page, { waitUntil: 'networkidle0' }),
-      possibleResults: getPossibleLoginResults(),
+      submitButtonSelector,
+      checkReadiness: async () => this.page.waitForSelector(loginSpinnerSelector, { hidden: true }),
```

Comment:
This is about the comment above. If `checkReadiness` is `Promise<void>`, this arrow function not returning `void`